src/gameGlobalInfo.cpp

```diff
@@ -244,6 +244,56 @@ int getSectorName(lua_State* L)
 /// Return the sector name for the point with coordinates (x, y). Compare SpaceObject:getSectorName().
 REGISTER_SCRIPT_FUNCTION(getSectorName);
 
+glm::vec2 sectorToXY(string sector)
+{
+    constexpr float sector_size = 20000;
+    int x, y, intpart;
+
+    // Y axis is complicated
+    if(sector[0] >= char('A') && sector[1] >= char('A')){
```

Comment:
This could crash if `sector.length() < 2`

Reply:
Good catch, thanks! I'll try to get a fix in this evening.

Reply:
Fix was tested with the following:
```
  PlayerSpaceship():setTemplate("Atlantis"):setPosition(sectorToXY("AA")):setCallSign("AA")
  PlayerSpaceship():setTemplate("Atlantis"):setPosition(sectorToXY("A")):setCallSign("A")
  PlayerSpaceship():setTemplate("Atlantis"):setPosition(sectorToXY("")):setCallSign("_blank_")
```

No crashes.